recipes/annotsv/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "annotsv" %}
+{% set version = "3.3.2" %}
+{% set sha256 = "f4dacc77cd6f31cad6fc00e8a07449eeaace4ca1c47a357fc6bc7f6d621615e3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/lgmgeo/AnnotSV/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - make
+  host:
+    - tk >=8.5
```

Comment:
I'm at a bit of a loss on the versions errors, puzzling, it's building locally (at least it would be if it weren't trying to download tons of data ;-) )

Reply:
You might have to ask on the Bioconda Gitter- sorry, a solution is not presenting itself to me right now.